package/MDAnalysis/transformations/translate.py

```diff
@@ -47,9 +50,7 @@ def translate(vector):
     ----------
     vector: list
```

Comment:
This is not supposed to be a list, but an array of anything similar. I think "array-like" is the term.